### PR TITLE
grpc gateway decoding

### PR DESCRIPTION
**Pull request by javierguerragiraldez**
structured URL args and timestamp transcoding

### CODE REVIEW

File: app/_hub/kong-inc/grpc-gateway/index.md [hunk 1]
```diff
@@ -160,6 +164,24 @@ curl -XPOST localhost:8000/v1/messages/Kong2.0 -d '{"name":"kong2.0"}'
 
 All syntax defined in [Path template syntax](https://github.com/googleapis/googleapis/blob/fc37c47e70b83c1cc5cc1616c9a307c4303fe789/google/api/http.proto#L225) is supported.
 
+Object fields not mentioned in the paths like `/messages/{name}` can be passed as URL arguments (like `/v1/messages?name=Kong`).  Structured arguments are supported; a request like
+
+`/v1/messages?dest.name=Freddy&dest.address=1428+Elm+Street&message=One+Two...`
+
+is interpreted like a JSON object:
```
Comment (HCloward) [inline app/_hub/kong-inc/grpc-gateway/index.md:171]:
```suggestion
is interpreted like the following JSON object:
```

File: app/_hub/kong-inc/grpc-gateway/index.md [hunk 2]
```diff
@@ -160,6 +164,24 @@ curl -XPOST localhost:8000/v1/messages/Kong2.0 -d '{"name":"kong2.0"}'
 
 All syntax defined in [Path template syntax](https://github.com/googleapis/googleapis/blob/fc37c47e70b83c1cc5cc1616c9a307c4303fe789/google/api/http.proto#L225) is supported.
 
+Object fields not mentioned in the paths like `/messages/{name}` can be passed as URL arguments (like `/v1/messages?name=Kong`).  Structured arguments are supported; a request like
```
Comment (HCloward) [inline app/_hub/kong-inc/grpc-gateway/index.md:167]:
```suggestion
Object fields not mentioned in the endpoint paths as in `/messages/{name}` can be passed as URL arguments, for example `/v1/messages?name=Kong`.  Structured arguments are supported.

 For example, a request like the following:
```

**Review by HCloward — COMMENTED** [review]
Hi @javierguerragiraldez - I'm not sure from the comment in this PR what this doc update is in reference to. It appears to be for v2.6.x but I don't see a plugin version update. Was the plugin code updated? If so why no version update. If the code was not affected and it is just a doc update, is there a reason for the update? a Jira or Trello issue from a user?  I just need some context to understand. Thanks!

**Comment by javierguerragiraldez** [comment]
> It appears to be for v2.6.x but I don't see a plugin version update. Was the plugin code updated?

Kong/kong#7538
Kong/kong#7564


> If so why no version update.

this plugin has been imported into the main repo, and now it's released with the rest of Kong I'm not sure if we'll keep counting version numbers for them.

**Comment by PaulFischer-Kong** [comment]
@HCloward Will not be merging this PR for the 2.6 release - we'll address this doc change in the coming 2.6.1 patch coming up.

**Comment by PaulFischer-Kong** [comment]
This is now fixed in the release for 2.6 paving the way for this doc.

**Comment by PaulFischer-Kong** [comment]
We are fixing this PR

**Review by kikito — APPROVED** [review]

**Pull request merged**